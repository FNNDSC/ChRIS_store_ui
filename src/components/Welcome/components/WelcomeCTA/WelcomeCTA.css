.welcome-cta {
  font-size: 1.4em;
}

.welcome-cta-img {
  /* color */
  background: #393f44;
  background: var(--pf-black-800);
  background: linear-gradient(135deg #4d5258, #393f44);
  background: linear-gradient(135deg, var(--pf-black-700), var(--pf-black-800));
  background: url('../../../../assets/img/banner_chris-users.png') #393f44;
  background-repeat: no-repeat;
  background-size: cover;
  background-position: center;
  color: #fafafa;
  color: var(--pf-black-100);

  /* display */
  padding: 3em 30px;

  /* font */
  text-shadow: 3px 3px 6px #030303;
  text-shadow: 3px 3px 6px var(--pf-black);

  /* vertically align text in div */
  display: flex;
  justify-content: center;
  flex-direction: column;
}

.welcome-cta-header {
  /* font */
  font-size: 1.3em;
  line-height: 1.3;
}

.welcome-cta-featured {
  /* color */
  background: #0088ce;
  background: var(--pf-blue-400);
  background: linear-gradient(#39a5dc, #0088ce);
  background: linear-gradient(var(--pf-blue-300), var(--pf-blue-400));

  /* display */
  min-height: 300px;
  padding-bottom: 1.5em;
}

.welcome-cta-featured-desc {
  /* color */
  color: #fafafa;
  color: var(--pf-black-100);

  /* display */
  padding: 1.5em 1em;

  /* font */
  text-align: center;
  font-weight: 300;
}

.welcome-cta .welcome-cta-featured-container {
  /* display */
  display: flex;
  width: 300px;
  width: fit-content;

  /* flex */
  flex-direction: column;
}

.welcome-cta-featured-container > * {
  flex: 1;
}

.welcome-scroll-caret {
  /* color */
  color: #fafafa;
  color: var(--pf-black-100);

  /* display */
  display: none;

  /* font */
  font-size: 2em;
  text-align: center;
}

.welcome-user-cta {
  /* display */
  padding: 6em 2em;
  padding-bottom: 8em;
}

.welcome-user-cta-header {
  /* display */
  margin-bottom: 0.5em;

  /* font */
  font-size: 2em;
}

/* ============================== */
/* ---------- ANIMATIONS -------- */
/* ============================== */

@keyframes fadein {
  0% { opacity: 0; }
  60% { opacity: 0; }
  100% { opacity: 1; }
}

/* ============================== */
/* ---------- RESPONSIVE -------- */
/* ============================== */

@media (min-width: 480px)  {
  .welcome-cta-img {
    /* display */
    padding: 3em 2em;
  }

  .welcome-cta-header {
    /* font */
    font-size: 1.9em;
  }
}

@media (min-width: 768px)  {
  .welcome-cta-img {
    /* display */
    min-height: 300px;
    /* fill up the entire page except for the features (300px) and navbar (66px) */
    height: calc(100vh - 325px - 66px);
    padding: 0;
  }

  .welcome-cta-header {
    /* font */
    font-size: 2.5em;
  }

  .welcome-cta-featured {
    /* display */
    min-height: 325px;
    padding-bottom: 0;
  }

  .welcome-cta .welcome-cta-featured-container {
    /* display */
    width: auto;

    /* flex */
    flex-direction: row;
  }

  .welcome-cta-header { padding: 0 8%; }

  .welcome-scroll-caret {
    /* display */
    display: block;

    /* other */
    animation: fadein 2s;
  }

  .welcome-user-cta {
    width: 70%;
  }
}
