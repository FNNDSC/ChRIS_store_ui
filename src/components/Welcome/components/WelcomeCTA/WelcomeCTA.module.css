.welcomeCta {
  font-size: 1.4em;

  /* color */
  /* background: #393f44;
  background: var(--pf-black-800);
  background: linear-gradient(135deg #4d5258, #393f44);
  background: linear-gradient(135deg, var(--pf-black-700), var(--pf-black-800)); */
  background: url('../../../../assets/img/banner_chris-users.png');
  background: linear-gradient(transparent, var(--pf-blue-600) 96%, var(--pf-blue-500) 100%),
    linear-gradient(transparent, #0b68b440, var(--pf-blue-300)), 
    url('../../../../assets/img/banner_chris-users.png');

  background-repeat: no-repeat;
  background-size: cover;
  background-position: left;
  background-attachment: fixed;
  color: #fafafa;
  color: var(--pf-black-100);

  
  /* vertically align text in div */
  display: flex;
  justify-content: center;
  flex-direction: column;
  padding: 1em;
}

#welcomeCtaHeader {
  margin: 2em 0;
  text-align: center;
  padding: 2em 0;
} #welcomeCtaHeader h1 {
  font-size: 5em !important;
  font-weight: 600 !important;
  line-height: 1.25;
  margin: 0.5em 0;
  color: white;
} #welcomeCtaHeader p {
  font-size: 1.5em !important;
  line-height: 1.25 !important;
  max-width: 600px;
}

.welcomeCta .welcomeCtaFeaturedContainer {
  display: flex;
  width: 300px;
  width: fit-content;

  /* flex */
  flex-flow: row;
  flex-wrap: wrap;
} .welcomeCtaFeaturedContainer > * {
  flex: 1;
}

/* ============================== */
/* ---------- ANIMATIONS -------- */
/* ============================== */

@keyframes fadein {
  0% { opacity: 0; }
  60% { opacity: 0; }
  100% { opacity: 1; }
}

/* ============================== */
/* ---------- RESPONSIVE -------- */
/* ============================== */

@media (min-width: 480px)  {
  .welcomeCtaHeader {
    font-size: 2em;
  }
}

@media (min-width: 768px)  {
  .welcomeCtaHeader {
    font-size: 2.5em;
    padding: 4em 0;
  }

  .welcomeCta .welcomeCtaFeatured {
    padding: 3em 0 6em 0;
  }
}
