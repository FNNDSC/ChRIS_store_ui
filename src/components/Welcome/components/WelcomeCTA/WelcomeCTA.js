import React from 'react';
import { Icon } from 'patternfly-react';
import './WelcomeCTA.css';
import WelcomeFeature from '../WelcomeFeature/WelcomeFeature';

/* plugin images */
import freesurferLogo from '../../../../assets/img/plugins/freesurfer_90.png';
import tensorflowLogo from '../../../../assets/img/plugins/tensorflow_90.png';
import fslLogo from '../../../../assets/img/plugins/fsl_90.png';
import rhoanaLogo from '../../../../assets/img/plugins/rhoana_90.png';
import slicerLogo from '../../../../assets/img/plugins/slicer_90.png';

const WelcomeCTA = () => (
  <div className="welcome-cta">
    <div className="welcome-cta-img">
      <div className="row welcome-cta-header">
        Accessible medical imaging using the latest
        research innovations, backed by cloud-based
        computing power.
      </div>
    </div>
    <div className="welcome-cta-featured">
      <div className="welcome-cta-featured-desc">
        Apps available in the ChRIS store include:
      </div>
      <div className="row welcome-cta-featured-container">
        <WelcomeFeature name="FreeSurfer" img={freesurferLogo} />
        <WelcomeFeature name="TensorFlow" img={tensorflowLogo} />
        <WelcomeFeature name="FMRIB Software Library" img={fslLogo} />
        <WelcomeFeature name="RhoANA" img={rhoanaLogo} />
        <WelcomeFeature name="Slicer" img={slicerLogo} />
      </div>
      <div className="welcome-scroll-caret">
        <Icon name="angle-down" />
      </div>
    </div>
    <div className="row no-flex">
      <div className="welcome-user-cta">
        <div className="welcome-user-cta-header">
          Focus on your data. Not the tools.
        </div>
        <div className="text-light">
          You need to run analyses on data, view the results,
          create visualizations, collaborate on your findings.
          Not build an infrastructure and become a software developer.
        </div>
        <br />
        <div className="text-light">
          The ChRIS platform provides a common infrastructure to
          which you can deploy...
        </div>
      </div>
    </div>
  </div>
);

export default WelcomeCTA;
