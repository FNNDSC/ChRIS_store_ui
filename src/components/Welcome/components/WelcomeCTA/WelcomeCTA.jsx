import React from 'react';
import './WelcomeCTA.css';
import WelcomeFeature from '../WelcomeFeature/WelcomeFeature';

/* plugin images */
import infantLogo from '../../../../assets/img/plugins/infant_90.png';
import freesurferLogo from '../../../../assets/img/plugins/freesurfer_90.png';
import biomediaLogo from '../../../../assets/img/plugins/biomedia_90.png';
import antLogo from '../../../../assets/img/plugins/ant_90.png';
import civetLogo from '../../../../assets/img/plugins/civet_90.png';

const WelcomeCTA = () => {
  // Object to make fetching from build/external easier later
  const features = [
    { name: 'Fastsurfer', url: 'https://chrisstore.co/plugin/44', img: freesurferLogo },
    { name: 'Civet', url: 'https://chrisstore.co/plugin/2', img: civetLogo },
    { name: 'Infant FreeSurfer', url: 'https://chrisstore.co/plugin/78', img: infantLogo },
    { name: 'IRTK Reconstruction', url: 'https://chrisstore.co/plugin/85', img: biomediaLogo },
    { name: 'N4 Bias Field Correction', url: 'https://chrisstore.co/plugin/77', img: antLogo },
  ];

  return (
    <>
      <div className="welcome-cta">
        <article>
          <div id="welcome-cta-header">
            <h1>ChRIS Store</h1>
            <p>
              Accessible medical imaging using the latest research innovations, backed by
              cloud-based computing power.
            </p>
          </div>
        </article>

        <article>
          <div className="welcome-cta-featured">
            {/* <div style={{ opacity: 0.75, color: 'white', textAlign: 'center' }}>
            <h2>Apps available in the ChRIS Store</h2>
          </div> */}
            <div className="row welcome-cta-featured-container">
              {features.map((feature) => (
                <WelcomeFeature key={feature.name} {...feature} />
              ))}
            </div>
          </div>
        </article>
      </div>
    </>
  );
};

export default WelcomeCTA;
