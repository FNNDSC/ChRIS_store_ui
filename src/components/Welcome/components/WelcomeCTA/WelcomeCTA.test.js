import React from 'react';
import { shallow } from 'enzyme';
import WelcomeCTA from './WelcomeCTA';

describe('WelcomeCTA', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<WelcomeCTA />);
  });

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should render welcome-cat-img div', () => {
    expect(wrapper.find('div.welcome-cta-img')).toHaveLength(1);
  });

  it('should render welcome-cta-header with row class', () => {
    expect(wrapper.find('div.welcome-cta-header').hasClass('row')).toEqual(true);
  });

  it('should render welcome-cta-featured div', () => {
    expect(wrapper.find('div.welcome-cta-featured')).toHaveLength(1);
  });

  it('should render welcome-cta-featured-container div', () => {
    expect(wrapper.find('div.welcome-cta-featured')).toHaveLength(1);
  });

  it('featured container should have row class', () => {
    expect(wrapper
      .find('div.welcome-cta-featured-container')
      .hasClass('row'))
      .toEqual(true);
  });

  it('should render 5 welcome features in the featured container', () => {
    expect(wrapper
      .find('div.welcome-cta-featured-container')
      .find('WelcomeFeature'))
      .toHaveLength(5);
  });

  it('should render welcome-scroll-caret div', () => {
    expect(wrapper.find('div.welcome-scroll-caret')).toHaveLength(1);
  });

  it('should render an Icon component inside of the scroll caret div', () => {
    expect(wrapper
      .find('div.welcome-scroll-caret')
      .find('Icon'))
      .toHaveLength(1);
  });

  it('should render row with welcome-user-cta div inside', () => {
    expect(wrapper
      .find('div.row > div.welcome-user-cta')
      .find('div.welcome-user-cta'))
      .toHaveLength(1);
  });

  it('should render welcome-user-cta-header div', () => {
    expect(wrapper
      .find('div.welcome-user-cta')
      .find('div.welcome-user-cta-header'))
      .toHaveLength(1);
  });

  it('should render 2 text-light divs inside the welcome user cta', () => {
    expect(wrapper
      .find('div.welcome-user-cta')
      .find('div.text-light'))
      .toHaveLength(2);
  });

  it('should render br element inside the welcome user cta', () => {
    expect(wrapper
      .find('div.welcome-user-cta')
      .find('br'))
      .toHaveLength(1);
  });
});
