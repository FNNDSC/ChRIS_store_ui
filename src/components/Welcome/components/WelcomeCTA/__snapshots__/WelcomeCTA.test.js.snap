// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomeCTA should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WelcomeCTA />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="welcome-cta-img"
>
          <div
                    className="row welcome-cta-header"
          >
                    Accessible medical imaging using the latest research innovations, backed by cloud-based computing power.
          </div>
</div>,
        <div
          className="welcome-cta-featured"
>
          <div
                    className="welcome-cta-featured-desc"
          >
                    Apps available in the ChRIS store include:
          </div>
          <div
                    className="row welcome-cta-featured-container"
          >
                    <WelcomeFeature
                              img="freesurfer_90.png"
                              name="FreeSurfer"
                    />
                    <WelcomeFeature
                              img="tensorflow_90.png"
                              name="TensorFlow"
                    />
                    <WelcomeFeature
                              img="fsl_90.png"
                              name="FMRIB Software Library"
                    />
                    <WelcomeFeature
                              img="rhoana_90.png"
                              name="RhoANA"
                    />
                    <WelcomeFeature
                              img="slicer_90.png"
                              name="Slicer"
                    />
          </div>
          <div
                    className="welcome-scroll-caret"
          >
                    <Icon
                              name="angle-down"
                              type="fa"
                    />
          </div>
</div>,
        <div
          className="row no-flex"
>
          <div
                    className="welcome-user-cta"
          >
                    <div
                              className="welcome-user-cta-header"
                    >
                              Focus on your data. Not the tools.
                    </div>
                    <div
                              className="text-light"
                    >
                              You need to run analyses on data, view the results, create visualizations, collaborate on your findings. Not build an infrastructure and become a software developer.
                    </div>
                    <br />
                    <div
                              className="text-light"
                    >
                              The ChRIS platform provides a common infrastructure to which you can deploy...
                    </div>
          </div>
</div>,
      ],
      "className": "welcome-cta",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="row welcome-cta-header"
>
            Accessible medical imaging using the latest research innovations, backed by cloud-based computing power.
</div>,
          "className": "welcome-cta-img",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Accessible medical imaging using the latest research innovations, backed by cloud-based computing power.",
            "className": "row welcome-cta-header",
          },
          "ref": null,
          "rendered": "Accessible medical imaging using the latest research innovations, backed by cloud-based computing power.",
          "type": "div",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="welcome-cta-featured-desc"
>
              Apps available in the ChRIS store include:
</div>,
            <div
              className="row welcome-cta-featured-container"
>
              <WelcomeFeature
                            img="freesurfer_90.png"
                            name="FreeSurfer"
              />
              <WelcomeFeature
                            img="tensorflow_90.png"
                            name="TensorFlow"
              />
              <WelcomeFeature
                            img="fsl_90.png"
                            name="FMRIB Software Library"
              />
              <WelcomeFeature
                            img="rhoana_90.png"
                            name="RhoANA"
              />
              <WelcomeFeature
                            img="slicer_90.png"
                            name="Slicer"
              />
</div>,
            <div
              className="welcome-scroll-caret"
>
              <Icon
                            name="angle-down"
                            type="fa"
              />
</div>,
          ],
          "className": "welcome-cta-featured",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Apps available in the ChRIS store include:",
              "className": "welcome-cta-featured-desc",
            },
            "ref": null,
            "rendered": "Apps available in the ChRIS store include:",
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <WelcomeFeature
                  img="freesurfer_90.png"
                  name="FreeSurfer"
/>,
                <WelcomeFeature
                  img="tensorflow_90.png"
                  name="TensorFlow"
/>,
                <WelcomeFeature
                  img="fsl_90.png"
                  name="FMRIB Software Library"
/>,
                <WelcomeFeature
                  img="rhoana_90.png"
                  name="RhoANA"
/>,
                <WelcomeFeature
                  img="slicer_90.png"
                  name="Slicer"
/>,
              ],
              "className": "row welcome-cta-featured-container",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "img": "freesurfer_90.png",
                  "name": "FreeSurfer",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "img": "tensorflow_90.png",
                  "name": "TensorFlow",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "img": "fsl_90.png",
                  "name": "FMRIB Software Library",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "img": "rhoana_90.png",
                  "name": "RhoANA",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "img": "slicer_90.png",
                  "name": "Slicer",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Icon
                name="angle-down"
                type="fa"
/>,
              "className": "welcome-scroll-caret",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "name": "angle-down",
                "type": "fa",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="welcome-user-cta"
>
            <div
                        className="welcome-user-cta-header"
            >
                        Focus on your data. Not the tools.
            </div>
            <div
                        className="text-light"
            >
                        You need to run analyses on data, view the results, create visualizations, collaborate on your findings. Not build an infrastructure and become a software developer.
            </div>
            <br />
            <div
                        className="text-light"
            >
                        The ChRIS platform provides a common infrastructure to which you can deploy...
            </div>
</div>,
          "className": "row no-flex",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="welcome-user-cta-header"
>
                Focus on your data. Not the tools.
</div>,
              <div
                className="text-light"
>
                You need to run analyses on data, view the results, create visualizations, collaborate on your findings. Not build an infrastructure and become a software developer.
</div>,
              <br />,
              <div
                className="text-light"
>
                The ChRIS platform provides a common infrastructure to which you can deploy...
</div>,
            ],
            "className": "welcome-user-cta",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Focus on your data. Not the tools.",
                "className": "welcome-user-cta-header",
              },
              "ref": null,
              "rendered": "Focus on your data. Not the tools.",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "You need to run analyses on data, view the results, create visualizations, collaborate on your findings. Not build an infrastructure and become a software developer.",
                "className": "text-light",
              },
              "ref": null,
              "rendered": "You need to run analyses on data, view the results, create visualizations, collaborate on your findings. Not build an infrastructure and become a software developer.",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "The ChRIS platform provides a common infrastructure to which you can deploy...",
                "className": "text-light",
              },
              "ref": null,
              "rendered": "The ChRIS platform provides a common infrastructure to which you can deploy...",
              "type": "div",
            },
          ],
          "type": "div",
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="welcome-cta-img"
>
            <div
                        className="row welcome-cta-header"
            >
                        Accessible medical imaging using the latest research innovations, backed by cloud-based computing power.
            </div>
</div>,
          <div
            className="welcome-cta-featured"
>
            <div
                        className="welcome-cta-featured-desc"
            >
                        Apps available in the ChRIS store include:
            </div>
            <div
                        className="row welcome-cta-featured-container"
            >
                        <WelcomeFeature
                                    img="freesurfer_90.png"
                                    name="FreeSurfer"
                        />
                        <WelcomeFeature
                                    img="tensorflow_90.png"
                                    name="TensorFlow"
                        />
                        <WelcomeFeature
                                    img="fsl_90.png"
                                    name="FMRIB Software Library"
                        />
                        <WelcomeFeature
                                    img="rhoana_90.png"
                                    name="RhoANA"
                        />
                        <WelcomeFeature
                                    img="slicer_90.png"
                                    name="Slicer"
                        />
            </div>
            <div
                        className="welcome-scroll-caret"
            >
                        <Icon
                                    name="angle-down"
                                    type="fa"
                        />
            </div>
</div>,
          <div
            className="row no-flex"
>
            <div
                        className="welcome-user-cta"
            >
                        <div
                                    className="welcome-user-cta-header"
                        >
                                    Focus on your data. Not the tools.
                        </div>
                        <div
                                    className="text-light"
                        >
                                    You need to run analyses on data, view the results, create visualizations, collaborate on your findings. Not build an infrastructure and become a software developer.
                        </div>
                        <br />
                        <div
                                    className="text-light"
                        >
                                    The ChRIS platform provides a common infrastructure to which you can deploy...
                        </div>
            </div>
</div>,
        ],
        "className": "welcome-cta",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="row welcome-cta-header"
>
              Accessible medical imaging using the latest research innovations, backed by cloud-based computing power.
</div>,
            "className": "welcome-cta-img",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Accessible medical imaging using the latest research innovations, backed by cloud-based computing power.",
              "className": "row welcome-cta-header",
            },
            "ref": null,
            "rendered": "Accessible medical imaging using the latest research innovations, backed by cloud-based computing power.",
            "type": "div",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="welcome-cta-featured-desc"
>
                Apps available in the ChRIS store include:
</div>,
              <div
                className="row welcome-cta-featured-container"
>
                <WelcomeFeature
                                img="freesurfer_90.png"
                                name="FreeSurfer"
                />
                <WelcomeFeature
                                img="tensorflow_90.png"
                                name="TensorFlow"
                />
                <WelcomeFeature
                                img="fsl_90.png"
                                name="FMRIB Software Library"
                />
                <WelcomeFeature
                                img="rhoana_90.png"
                                name="RhoANA"
                />
                <WelcomeFeature
                                img="slicer_90.png"
                                name="Slicer"
                />
</div>,
              <div
                className="welcome-scroll-caret"
>
                <Icon
                                name="angle-down"
                                type="fa"
                />
</div>,
            ],
            "className": "welcome-cta-featured",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Apps available in the ChRIS store include:",
                "className": "welcome-cta-featured-desc",
              },
              "ref": null,
              "rendered": "Apps available in the ChRIS store include:",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <WelcomeFeature
                    img="freesurfer_90.png"
                    name="FreeSurfer"
/>,
                  <WelcomeFeature
                    img="tensorflow_90.png"
                    name="TensorFlow"
/>,
                  <WelcomeFeature
                    img="fsl_90.png"
                    name="FMRIB Software Library"
/>,
                  <WelcomeFeature
                    img="rhoana_90.png"
                    name="RhoANA"
/>,
                  <WelcomeFeature
                    img="slicer_90.png"
                    name="Slicer"
/>,
                ],
                "className": "row welcome-cta-featured-container",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "img": "freesurfer_90.png",
                    "name": "FreeSurfer",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "img": "tensorflow_90.png",
                    "name": "TensorFlow",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "img": "fsl_90.png",
                    "name": "FMRIB Software Library",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "img": "rhoana_90.png",
                    "name": "RhoANA",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "img": "slicer_90.png",
                    "name": "Slicer",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Icon
                  name="angle-down"
                  type="fa"
/>,
                "className": "welcome-scroll-caret",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "name": "angle-down",
                  "type": "fa",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="welcome-user-cta"
>
              <div
                            className="welcome-user-cta-header"
              >
                            Focus on your data. Not the tools.
              </div>
              <div
                            className="text-light"
              >
                            You need to run analyses on data, view the results, create visualizations, collaborate on your findings. Not build an infrastructure and become a software developer.
              </div>
              <br />
              <div
                            className="text-light"
              >
                            The ChRIS platform provides a common infrastructure to which you can deploy...
              </div>
</div>,
            "className": "row no-flex",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <div
                  className="welcome-user-cta-header"
>
                  Focus on your data. Not the tools.
</div>,
                <div
                  className="text-light"
>
                  You need to run analyses on data, view the results, create visualizations, collaborate on your findings. Not build an infrastructure and become a software developer.
</div>,
                <br />,
                <div
                  className="text-light"
>
                  The ChRIS platform provides a common infrastructure to which you can deploy...
</div>,
              ],
              "className": "welcome-user-cta",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Focus on your data. Not the tools.",
                  "className": "welcome-user-cta-header",
                },
                "ref": null,
                "rendered": "Focus on your data. Not the tools.",
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "You need to run analyses on data, view the results, create visualizations, collaborate on your findings. Not build an infrastructure and become a software developer.",
                  "className": "text-light",
                },
                "ref": null,
                "rendered": "You need to run analyses on data, view the results, create visualizations, collaborate on your findings. Not build an infrastructure and become a software developer.",
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": "br",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "The ChRIS platform provides a common infrastructure to which you can deploy...",
                  "className": "text-light",
                },
                "ref": null,
                "rendered": "The ChRIS platform provides a common infrastructure to which you can deploy...",
                "type": "div",
              },
            ],
            "type": "div",
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
